sdk/communication/Azure.Communication.Calling.Server/src/CallClient.cs

```diff
@@ -396,25 +396,24 @@ public virtual async Task<Response> InviteParticipantsAsync(string callLegId, IE
             }
         }
 
-        /// <summary> Invite participants to a call. </summary>
+        /// <summary> Add a participant to the call. </summary>
         /// <param name="callLegId"> The call leg id. </param>
-        /// <param name="participants"> The uri of the audio file. </param>
-        /// <param name="alternateCallerId">The flag to indicate if audio file need to be played in a loop.</param>
+        /// <param name="participant"> The identity of participant to be added to the call. </param>
         /// <param name="operationContext">The operation context. </param>
-        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        /// <param name="alternateCallerId">The phone number to use when adding a pstn participant.</param>
+        /// <param name="cancellationToken"> The cancellation token. </param>
         /// <exception cref="RequestFailedException">The server returned an error. See <see cref="Exception.Message"/> for details returned from the server.</exception>
-        /// <exception cref="ArgumentNullException"> <paramref name="participants"/> is null. </exception>
-        public virtual Response InviteParticipants(string callLegId, IEnumerable<CommunicationIdentifier> participants, string alternateCallerId = default, string operationContext = null, CancellationToken cancellationToken = default)
+        public virtual Response AddParticipant(string callLegId, CommunicationIdentifier participant, string operationContext, string alternateCallerId = default, CancellationToken cancellationToken = default)
         {
-            using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(CallClient)}.{nameof(InviteParticipants)}");
+            using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(CallClient)}.{nameof(AddParticipant)}");
             scope.Start();
             try
             {
-                Argument.AssertNotNullOrEmpty(participants, nameof(participants));
+                Argument.AssertNotNull(participant, nameof(participant));
                 Argument.AssertNotNullOrEmpty(alternateCallerId, nameof(alternateCallerId));
 
                 return RestClient.InviteParticipants(callLegId,
-                    participants.Select(p => CommunicationIdentifierSerializer.Serialize(p)),
+                    new List<CommunicationIdentifierModel>() { CommunicationIdentifierSerializer.Serialize(participant) },
```

Comment:
I don't why we need to have a CommunicationIdentifierModel List pass to Rest Client.
We Invite participant one at a time, if so, I think we should just use CommunicationIdentifierModel .

Reply:
[4:01 PM] Naveed Ali api/pma right now support list 
[4:02 PM] Naveed Ali but we want to restrict to 1 on sdk level